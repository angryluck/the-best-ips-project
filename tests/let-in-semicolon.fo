fun int main() =
    let tmp1 = write(let x = 2; y = 3 in x+y) in
    let tmp2 = write(let a = 3; b = 10 in a*b) in 
    let tmp3 = write(let x = 4; y = x + 1 in y*2) in
    let tmp4 = write(let a = true; b = if a then 1 else 0 in b + 5) in
    0 