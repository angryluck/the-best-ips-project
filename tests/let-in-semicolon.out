5
30
10
6